db/sql-playground.sql

```diff
@@ -0,0 +1,6 @@
+SELECT reviews.*, COUNT(comments.review_id) AS comment_count
```

Comment:
I think this is a test file? To check your SQL requests? If so, don't include it in your PRs as it's not relevant to the ticket